src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/dashboard/dashboard.component.html

```diff
@@ -1,4 +1,5 @@
-<div>
+<main>
+  <h1 class="sr-only">Dashboard</h1>
```

Comment:
At this point you're far more experienced on a11y than myself, but what about not adding structural elements for screen readers alone, and just tag the existing ones? Would this still meet A level?
```suggestion
<main aria-label="Dashboard" i18n-aria-label aria-description="This page presents a summary of blah blah" i18n-aria-description>
```

Reply:
@epuertat the need for the H1 is reported by axe when the e2e test runs. The page doesn't have any top level heading

![image](https://user-images.githubusercontent.com/15856785/182616803-68836d59-6899-4ec6-8bd9-aa2c464d2532.png)

Attempting to use aria labels does work as explained [here](https://www.w3.org/WAI/tutorials/page-structure/labels/#using-aria-labelledby) but still triggers the axe warnings. Another option could be to make the  heading visible to all users like below that way we can also use it as a label for the main element

![image](https://user-images.githubusercontent.com/15856785/182630120-a54dba6e-f708-4dbc-9715-58d414260502.png)

Reply:
Now I get it. Thanks for the explanation. The truth is that we're no using the `<h1>` anywhere in the Dashboard (only in the login page and that was, I guess, recently added by you).

I've been trying to find the WCAG rule that established this, but I could only [this one from deque folks](https://dequeuniversity.com/rules/axe/4.3/page-has-heading-one). It seems that the rationale of this is that it allows screen readers to directly skip all the top/side navigation stuff and jump right into the main content. So far that purpose, I think [the following approach could also work](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role):

```suggestion
<main role="heading" aria-level="1">
```
It the purpose is to provide screen readers an anchor to "where the real stuff lies", I guess we should put that `role="heading aria-level="1" ...>" in the outlet template. That way all the pages that use the same outlet template would have this `h1` anchor. What do you think?

Reply:
Sure, will give it a try. We will have to change main to a div though, since it doesn't support the role=heading attribute. Not sure if we could achieve the same with the router outlet in the main app component. Will try that too

Reply:
@nSedrickm @epuertat another option would be to make the header invisible, i.e. making the text color the same as the background. Since headers are so well-vetted for screen readers, I feel like it would make the most sense to stick with what we know works. But if we don't want the heading to be visible, that would be fine as long as the screen reader would have something to jump to.

Reply:
The `sr-only` class should do that, but that will still require adding a header to each Dashboard page, so if there's a smarter/quicker way to do that I'm all for it.

Reply:
@nSedrickm instead of adding a heading, it would also be acceptable to add an invisible link to the top of the page that screen readers can use to skip to main content. See https://www.w3.org/TR/WCAG20-TECHS/G1.

Reply:
Hi @epuertat. I didn't  in making the axe a11y violation pass with the suggestions above. I also tried setting the role="heading" and aria-level="1" at various sections including the router outlet in an effort to have it accessible to all the dashboard pages but it didn't work. 

I finally went with @ljflores suggestion to add a skip to content link and also disabled the heading check in the axe testing rules